src/Xamarin.Android.Build.Tasks/Tasks/Aot.cs

```diff
@@ -366,13 +366,13 @@ IEnumerable<Config> GetAotConfigs ()
 
 					var libs = new List<string>();
 					if (NdkUtil.UsingClangNDK) {
-						libs.Add ($"-L{toolchainLibDir}");
-						libs.Add ($"-L{androidLibPath}");
+						libs.Add ($"-L\\\"{toolchainLibDir}\\\"");
```

Comment:
This should instead be `toolchainLibDir.TrimEnd('\\')`.  Ditto everywhere else.

Reply:
Also, would this work if the quotes were on the "outside", e.g. `$"\\\"-L{toolchainLibDir.TrimEnd('\\')\\\"")`.